TravelerApp/TravelerAppCore/Controller/AddHotelInfo.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
```

Comment:
w kodzie powinien być porządek.  Powinny znajdować się w nim tylko potrzebne rzeczy. Dlatego usuń wszystkie zbędne linie jak np nie używane usingi :) zastosuj do wszystkich plików gdzie to jest potrzebne :)

Reply:
Posprzątane :)